modules/consensus/accept.go

```diff
@@ -154,6 +222,7 @@ func (cs *ConsensusSet) managedAcceptBlock(b types.Block) error {
 			if err == errFutureTimestamp {
 				go func() {
 					time.Sleep(time.Duration(b.Timestamp-(types.CurrentTimestamp()+types.FutureThreshold)) * time.Second)
+					// TODO: log error returned by AcceptBlock if non-nill?
```

Comment:
Would be easy if the consensus had logging, but since it doesn't yet, adding it should be another PR.